src/test/java/io/sentry/integration/autoinstall/log4j2/Log4jAutoInstallTestIT.kt

```diff
@@ -0,0 +1,117 @@
+package io.sentry.autoinstall.log4j2
+
+import basePom
+import installMavenWrapper
+import io.sentry.autoinstall.SentryInstaller
+import org.apache.maven.shared.verifier.VerificationException
+import org.apache.maven.shared.verifier.Verifier
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.StandardOpenOption
+import kotlin.io.path.Path
+
+class Log4jAutoInstallTestIT {
+    @TempDir()
+    lateinit var file: File
+
+    @BeforeEach
+    fun installWrapper() {
+        installMavenWrapper(file, "3.8.6")
+    }
+
+    fun getPOM(
+        installLog4j2: Boolean = true,
+        log4j2Version: String = "2.17.0",
+        sentryLog4j2Version: String = "6.25.2",
+        installedSentryVersion: String? = null,
+    ): String {
+        var dependencies =
+            """
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-api</artifactId>
+                <version>$log4j2Version</version>
+            </dependency>
+            """.trimIndent()
+
+        if (!installLog4j2) {
+            dependencies =
+                dependencies.plus(
+                    """
+                    <dependency>
+                        <groupId>io.sentry</groupId>
+                        <artifactId>sentry-log4j2</artifactId>
+                        <version>$sentryLog4j2Version</version>
+                    </dependency>
+                    """.trimIndent(),
+                )
+        }
+
+        val pomContent = basePom(dependencies, installedSentryVersion)
+
+        Files.write(Path("${file.absolutePath}/pom.xml"), pomContent.toByteArray(), StandardOpenOption.CREATE)
+
+        return file.absolutePath
+    }
+
+    @Test
+    @Throws(VerificationException::class, IOException::class)
+    fun `when sentry-log4j2 is a direct dependency logs a message and does nothing`() {
+        val path = getPOM(false)
+        val verifier = Verifier(path)
+        verifier.isAutoclean = false
+        verifier.addCliArgument("install")
+        verifier.execute()
+        verifier.verifyTextInLog("sentry-log4j2 won't be installed because it was already installed directly")
+        verifier.resetStreams()
+    }
+
+//    @Test
```

Comment:
Removed from Integration Tests, there's a unit test for it and there is no version < 2.0.0 of log4j available on maven central, thus the test would fail because it can't resolve the dep.